packages/deploy-script-support/src/coreProposalBehavior.js

```diff
@@ -54,37 +54,55 @@ export const makeCoreProposalBehavior = ({
     return fromEntries(ents);
   };
 
-  /** @param {ChainBootstrapSpace & BootstrapPowers & { evaluateInstallation: any }} allPowers */
+  /** @param {ChainBootstrapSpace & BootstrapPowers & { evaluateBundleCap: any }} allPowers */
   const behavior = async allPowers => {
     const {
-      consume: { board, agoricNamesAdmin },
-      evaluateInstallation,
+      consume: { vatAdminSvc, zoe, agoricNamesAdmin },
+      evaluateBundleCap,
       installation: { produce: produceInstallations },
       modules: {
         utils: { runModuleBehaviors },
       },
     } = allPowers;
     const [exportedGetManifest, ...manifestArgs] = getManifestCall;
 
-    const restoreRef = overrideRestoreRef || (x => E(board).getValue(x));
+    const defaultRestoreRef = async args => {
+      let p;
+      if (args.bundleName) {
+        p = E(vatAdminSvc).getNamedBundleID(args.bundleName);
+      } else {
+        p = Promise.resolve(args.bundleID);
+      }
+      const bundleID = await p;
+      return E(zoe).installBundleID(bundleID);
+    };
+    const restoreRef = overrideRestoreRef || defaultRestoreRef;
 
     // Get the on-chain installation containing the manifest and behaviors.
-    console.info('restoreRef, evaluateInstallation', {
+    console.info('restoreRef, evaluateBundleCap', {
       manifestInstallRef,
       exportedGetManifest,
+      vatAdminSvc,
     });
-    const manifestInstallation = await restoreRef(manifestInstallRef);
```

Comment:
Did `restoreRef` go away altogether? I need to understand that better.

gotta pause this for now...

Reply:
no, `restoreRef` is still passed to the code that creates zoe installations from bundlerefs, it just isn't used to install that code itself